### ISSUE TITLE
Python 2.7 support

### ISSUE THREAD

**Issue by pszpetkowski** [sunscrapers/django-rest-payments#1]
Should we support it or should we just go for 3.x?

I suppose that both decisions have their own pros and cons, however recently there is a trend to slowly deprecate Python 2 within Python/Django community (e.g. tomchristie/django-rest-framework#4799), however Django 1.11 LTS will keep supporting it at least until April 2020, which might also be extended.

Supporting Python 2 might also bring more dependencies into project and would certainly require writing more code to handle compatibility cases. On the other hand - more people would be able to make use of the project, however does it really matter that much right now? Back in 2008 when Python 3 came out it would be quite obvious that supporting Python 2.x is a must, however today lots of projects prioritize Python 3 and even ignore 2.7 altogether.

**Comment by pawelswiecki** [comment]
I don't think we need to support Python 2. I think this lib won't be used by old projects, since adding simple payments to 5 yo project is very unlikely. So to keep development and testing costs down I'd go for Python 3.4+.

Also, I feel somewhat moral responsibility to push for Python 3 ;)

**Comment by auvipy** [comment]
python3 FTW
